tools/challenge-helper-scripts/helpers/get-step-template.js

```diff
@@ -27,7 +27,10 @@ function getStepTemplate({
 }) {
   const seedTexts = Object.values(challengeSeeds)
     .map(({ contents, ext, editableRegionBoundaries }) => {
-      const fullContents = insertErms(contents, editableRegionBoundaries);
+      let fullContents = contents;
+      if (editableRegionBoundaries.length >= 2) {
```

Comment:
```suggestion
      if (editableRegionBoundaries.length === 2) {
```
Nitpicky, I know, but it currently does have to be exactly 2.

Reply:
I thought the idea was to allow it, but ignore it: https://github.com/freeCodeCamp/freeCodeCamp/pull/42903/commits/e46fa3099dcb07c445e3b1ceb867b43556bf359a#diff-785b651b36e382407def34a0ca4d23fbad88db34d2611924ecc572047ce86b18R32

Reply:
I think we want to know if the number of erms is wrong. It's not a big deal either way, so we can merge as is if you like.